app/witness.gradle

```diff
@@ -84,6 +84,7 @@ dependencyVerification {
         'com.android.tools.analytics-library:shared:27.1.3:shared-27.1.3.jar:10d2a51d8f89ff4ac849888e5a9c60b10e879c30d78545ec1da4d3df7bd56ae4',
         'com.android.tools.analytics-library:tracker:27.1.3:tracker-27.1.3.jar:589b355a2ba796cbc0a2b2295737de6661f078262e5f87cd6f540b8d011e5ebb',
         'com.android.tools.build:aapt2-proto:4.1.0-alpha01-6193524:aapt2-proto-4.1.0-alpha01-6193524.jar:17e75523e1e92dd4f222c7368ee41df9e964a508232f591e265d0c499baf9dca',
+        'com.android.tools.build:aapt2:4.1.3-6503028:aapt2-4.1.3-6503028-osx.jar:6c57e78049706e57ff65ad8c5fef43ed840c086710aeafa701953d6abbc90724',
```

Comment:
This is problematic as it is platform-dependent. Will have to remove that before merging. If #759 gets merged first, that should be no problem.

Reply:
Is this dependency needed?